apertium/pretransfer.h

```diff
@@ -2,12 +2,7 @@
 #define PRETRANSFER_H
 
 #include <iostream>
```

Comment:
It appears this file is also missing the file copyright header. Should be a quick fix. My bad for not mentioning it earlier.

Reply:
should I add 2019 as the year? https://github.com/apertium/apertium/blob/master/apertium/interchunk.h#L2

Reply:
Eh, copyright notice years are a controversial topic. I don't really care much about them.